packages/ses/src/make-eval-function.js

```diff
@@ -19,7 +23,30 @@ export const makeEvalFunction = safeEvaluate => {
         // rule. Track.
         return source;
       }
-      return safeEvaluate(source);
+      if (legacyHermesTaming === 'unsafe') {
+        throw TypeError(
+          `Legacy Hermes unsupported eval() with strings arguments cannot be tamed safe under legacyHermesTaming ${legacyHermesTaming}
```

Comment:
```suggestion
          `Legacy Hermes unsupported eval() called with strings arguments cannot be tamed safe under legacyHermesTaming ${legacyHermesTaming}
```

Reply:
also needs to throw on makeFunctionConstructor to cover all cases

Reply:
throw earlier instead

Reply:
consider throwing earlier instead at global-object.js (src/lockdown.js too early)

Reply:
under safeEval we want run makeSafeEvaluator on no-direct-eval hosts that support the with statement
(e.g. yet-another-es-engine, static hermes [with statement](https://github.com/facebook/hermes/pull/1571) backported to legacy hermes)
i.e. try makeSafeEvaluator on string, runs transparently if `with` supported, otherwise throw TypeError